geoips/plugins/modules/readers/windsat_idr37_binary.py

```diff
@@ -341,11 +347,12 @@ def call(fnames, metadata_only=False, chans=None, area_def=None, self_register=F
                 0
             ]  # sec since 1200Z,01/01/2000
             # get time info for each data point
+            # fmt: off
             timeinfo = pd.datetime(2000, 1, 1, 12) + pd.Timedelta(jd2000, unit="s")
-            time_date = int(
+            time_date = int(  # NOQA
                 str(timeinfo.year) + str(timeinfo.month) + str(timeinfo.day)
             )
-            time_hhmm = int(str(timeinfo.hour) + str(timeinfo.minute))
+            time_hhmm = int(str(timeinfo.hour) + str(timeinfo.minute))  # NOQA
```

Comment:
These can probably just be commented out if unused?  Not aprt of the binary read?